tools/make-cert-key.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+week=$(date "+%V")
+((num=$week - $week % 2))
+year_month=$(date "+%Y_%m")
```

Comment:
I'd definitely remove the month from here, it contributes nothing, and we can have weeks being cut by a new month, and then on Tuesday we have one key and on Wednesday a different key, and therefore, a different cache being built.